### DISCUSSION TITLE
First Community Meeting 28/04/2022

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#6574] [General]
Hello to all Arduino ESP32 Core Users,

we appreciate all of your contribution and help and we would like to improve your User experience on this Project. :star_struck:

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfill this, we would like to **held every Month Community Meetings** where we can share main updates and also have an open discussion with you.

This meeting is the first one so we have some suggestions for Meeting Agenda but there is still a lot of open space for your topics and questions so please share them in the comments below.

:boom:**Meeting will take place on Thursday 28/04/2022 at 4 pm GMT+2 time zone.** (1 hour length) We will use Google Meet.

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation. 

:innocent: If you want to be added to upcoming Meeting and receive invitation please send an email to vojtech.bartoska@espressif.com.

### 1) Espressif Introduction
- Community Meetings purpose
- Proposed topics process
- Brief Team introduction
- Issues tracker cleanup
- Explaining Sprint Meetings notes
- Project public Roadmap

### 2) Development updates
- summary of our current development

#### Documentation (proposed by @Xylopyrographer)
- explaining our approach and current Roadmap
- how we document APIs and why
- missing sections (what can be added)

### 4) Open Discussions
- place for any topic which is not in Agenda

See you on the Meeting!

**Comment by Xylopyrographer** [comment]
It would be good to prioritize documentation of the ESP32 versions of the Arduino API's. A number of the ESP32 versions are supersets of the stock Arduino ones documented on their site. WiFi, Bluetooth, I2C, Timers, are ones that come to mind.

Just as importantly is the need to document the sequence of calls to achieve a desired result. Things like what needs to be set up before a .begin() call? What can be changed after a .begin()?

Pin assignments. How does one set up a bus based customer pins? Or clock frequency? What is the correct sequence of calls to initiate? How do I know if the IO MUX has set things up to be hardware configured? Things like that.

**Reply by VojtechBartoska** [reply]
Valuable topic to the discussion, thanks for suggestion @Xylopyrographer. 

cc @pedrominatel

**Comment by VojtechBartoska** [comment]
Hello all,

Meeting Agenda is finished.

There was also one topic suggestion about **Regulatory compliance for commercial ESP32 usage (EMC, FCC, EU certificates)** and we sorted this topic to be answered only by email as it doesn't fit into the Meeting scope.

**Comment by VojtechBartoska** [comment]
**Summary of topics from Open discussion:**
### OTA encryption
- can be covered by PlatformIO IDE

### Debugging
- missing Documentation on Arduino ESP32 side
- investigate new Debugger in Arduino IDE
- can be also improved on PlatformIO end

### Commercial use of Arduino ESP32 Core
- typical User stories
- page with Licenses overview

**Reply by VojtechBartoska** [reply]
cc @pedrominatel, @ivankravets, @igrr

**Comment by VojtechBartoska** [comment]
Hello,

I'm sharing slides from the Presentation. Espressif Introduction slides can be helpful :)

[First_Community_meeting-Arduino-April_2022_GH.pdf](https://github.com/espressif/arduino-esp32/files/8610739/First_Community_meeting-Arduino-April_2022_GH.pdf)
